package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "b4a": "^1.6.1",
     "mutexify": "^1.4.0",
-    "streamx": "^2.12.4"
+    "streamx": "^2.13.2"
```

Comment:
Libraries like `hyperdrive-next` that uses `hyperblobs` are using latest `streamx`, so the deadlock appears on `hyperdrive-next`